docs/getting_started/custom-airflow-properties.rst

```diff
@@ -0,0 +1,23 @@
+Airflow Configuration Overrides with Astronomer Cosmos
+======================================================
+
+**Astronomer Cosmos** allows you to override Airflow configurations for each dbt task (dbt operator) via the dbt YAML file.
+
+### Sample dbt Model YAML
+
+.. code-block:: yaml
+
+    version: 2
+    models:
+      - name: name
+        description: description
+        meta:
+          cosmos:
+            pool: abcd
+
+### Explanation
+
+By adding Airflow configurations under ***cosmos*** in the ***meta*** field, you can set independent Airflow configurations for each task.
+For example, in the YAML above, the ***pool*** setting is applied to the specific dbt task.
+
```

Comment:
@wornjs thank you very much for adding the docs! What do you think if you extended the description by reusing what you wrote in the PR? The screenshots are very valuable